_core/smart/tutorial.md

```diff
@@ -3,306 +3,207 @@ layout: core
+    Take note of the `contracts` and `test` folders. The other files are boilerplate to wire up the project.
 
-Take a moment to `cat` the contents of the `sample-programs/names.clar` file.
-
-```bash
-cat sample-programs/names.clar
-````
+## Step 2: Reviewing hello world contract
 
-Which `tokens.clar` function is being called? 
+Now, let's have a look at a Clarity smart contract and get familiar with the basic language design characteristics.
 
-## Task 3: Initialize data-space and launch contracts
-
-In this task, you interact with the the contracts using the `clarity-cli` command line. 
-
-1. Initialize a new `db` database in the `/data/` directory
+1. Still inside the terminal, list the contents of the `contracts/sample` folder.
 
     ```bash
-    #  clarity-cli initialize /data/db
-    Database created
+    ls contracts/sample
     ```
 
-    You should see a message saying `Database created`. The command creates an SQLlite database.  The database is available in the container and also in your workstation. In this tutorial, your workstation mount should, at this point, contain the `$HOME/blockstack-dev-data/db`  directory.
+    This directory contains a hello world Clarity contract. Note that all Clarity files have a `.clar` suffix.
 
-2. Type check the `names.clar` contract.
+2. Let's review the contents of `hello-world.clar` with the `cat` command.
 
     ```bash
-    #  clarity-cli check sample-programs/names.clar /data/db
+    cat contracts/sample/hello-world.clar
     ```
-    
-    You should get an error:
 
-    ```
-    Error (line 11, column 1): use of unresolved contract ''S1G2081040G2081040G2081040G208105NK8PE5.tokens'.
-    ```
+    You should see the contract source code. Take a few seconds to review the content.
 
-    This happens because the `names.clar` contract _calls_ the `tokens.clar` contract, and that contract has not been created on the blockchain.
+    Clarity is a programming language based on [LISP](https://en.wikipedia.org/wiki/Lisp_(programming_language)). Most notably, Clarity is designed for static analysis, not compiled, and **not** [Turing complete](https://en.wikipedia.org/wiki/Turing_completeness).
 
-3. Type check the `tokens.clar` contract, it should pass a check as it does not use the `contract-call` function:
+    Let's go through the source code. Notice how the program and each statement is enclosed in `()` (parentheses). You'll see that the smart contract consists of two public methods. Starting at the top, let's review line by line:
 
-    ```bash
-    # clarity-cli check sample-programs/tokens.clar /data/db
-    Checks passed.
+    ```cl
+    (define-public (say-hi)
```

Comment:
I know we don't have any style guides for Clarity, but I'm not a fan of this indentation. I think this is much more readable:

```racket
(define-public (say-hi)
  (ok "hello world")
)
```

Mainly for functions longer than one line